test/units/plugins/action/test_synchronize.py

```diff
@@ -246,3 +250,15 @@ def test_delegate_remote_su(self):
         # delegate to other remote host with su enabled
         x = SynchronizeTester()
         x.runtest(fixturepath=os.path.join(self.fixturedir, 'delegate_remote_su'))
+
+    @patch.object(ActionModule, '_low_level_execute_command', side_effect=BreakPoint)
```

Comment:
... or is that just to get it to bail out on that call?

Reply:
I'm short circuiting the execution of `run` by setting a `side_effect` on a function call that will actually connect to the server to expand the path.  Otherwise it will try and go further, actually executing the module.

This allows me to kill execution immediately after what we are wanting to inspect.

Reply:
> ... or is that just to get it to bail out on that call?

Exactly that.

Reply:
Cool